### PR TITLE
RHDEVDOCS-5390: Documenting Pipelines 1.10.4 RN

### PR DESCRIPTION
**Pull request by shipsing**
**Purpose**: To resolve the following issue: 
* https://issues.redhat.com/browse/RHDEVDOCS-5390

**Aligned team**: DevTools

**OCP version this PR applies to (cherrypicking)**: enterprise 4.11, Enterprise 4.12, enterprise 4.13, enterprise 4.14

**Content for preview**:  https://61292--docspreview.netlify.app/openshift-enterprise/latest/cicd/pipelines/op-release-notes.html#release-notes-1-10-4_op-release-notes

**SME review** :  @vdemeester @PuneetPunamiya 

**QE review**:  @VeereshAradhya @ppitonak 

**Peer review**: @max-cx

### CODE REVIEW

**Review by vdemeester — APPROVED** [review]

File: modules/op-release-notes-1-10.adoc [hunk 1]
```diff
@@ -197,4 +197,18 @@ Before this update, the Tekton Operator did not expose the performance configura
...
+
+[id="release-notes-1-10-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.4
+
+With this update, {pipelines-title} General Availability (GA) 1.10.4 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-10-4_{context}"]
+=== Fixed issues
+
+* This update fixes the bundle conversion issue for `v1beta1` pipeline references. Now, the bundle conversion feature converts the `PipelineRef` bundle to the `Resolver` bundle.
```
Comment (piyush-garg) [inline modules/op-release-notes-1-10.adoc:210]: This update fixes the bundle resolver conversion issue for pipelineref field in PipelineRun. Now, the bundle conversion converts the `PipelineRef` bundle to the `Pipeline` kind in PipelineRun

**Review by ppitonak — APPROVED** [review]

File: modules/op-release-notes-1-10.adoc (hunk 1 above)
Comment (piyush-garg) [reply modules/op-release-notes-1-10.adoc:210]: This update fixes the bundle resolver conversion issue for pipelineref field in PipelineRun. Now, it sets the kind field to `Pipeline` after conversion

**Review by concaf — APPROVED** [review]

File: modules/op-release-notes-1-10.adoc (hunk 1 above)
Comment (vdemeester) [reply modules/op-release-notes-1-10.adoc:210]: yeah it's was setting the kind field to `Task` before (even when resolving Pipeline).. which made thing act weird to not working in some cases.

Comment (shipsing) [reply modules/op-release-notes-1-10.adoc]: done

File: modules/op-release-notes-1-10.adoc [hunk 2]
```diff
@@ -197,4 +197,18 @@ Before this update, the Tekton Operator did not expose the performance configura
...
+== Release notes for {pipelines-title} General Availability 1.10.4
+
+With this update, {pipelines-title} General Availability (GA) 1.10.4 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-10-4_{context}"]
+=== Fixed issues
+
+* This update fixes the bundle resolver conversion issue for the `PipelineRef` field in a pipeline run. Now, the conversion feature sets the value of the `kind` field to `Pipeline` after conversion. 
+
+* Before this update, the `pipelinerun.timeouts` field was reset to `timeouts.pipeline` value, ignoring `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.
```
Comment (max-cx) [inline modules/op-release-notes-1-10.adoc:212]:
```suggestion
* Before this update, the `pipelinerun.timeouts` field was reset to the `timeouts.pipeline` value, ignoring the `timeouts.tasks` and `timeouts.finally` values. This update fixes the issue and sets the correct default timeout value for a `PipelineRun` resource.
```

File: modules/op-release-notes-1-10.adoc [hunk 3]
```diff
@@ -197,4 +197,18 @@ Before this update, the Tekton Operator did not expose the performance configura
...
+
+* Before this update, the controller logs were spammed with unnecessary data. This update fixes the issue.
```
Comment (max-cx) [inline modules/op-release-notes-1-10.adoc:214]: Nitpick, to make it read more positive:
```suggestion
* Before this update, the controller logs contained unnecessary data. This update fixes the issue.
```

**Review by max-cx — APPROVED** [review]
I checked the preview.
I left two suggestions, which are up to the author to consider.

**Review by khrm — COMMENTED** [review]
We should merge this because release is out.

File: modules/op-release-notes-1-10.adoc [hunk 4]
```diff
@@ -197,4 +197,18 @@ Before this update, the Tekton Operator did not expose the performance configura
...
+
+[id="release-notes-1-10-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.4
+
+With this update, {pipelines-title} General Availability (GA) 1.10.4 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-10-4_{context}"]
+=== Fixed issues
+
+* This update fixes the bundle resolver conversion issue for the `PipelineRef` field in a pipeline run. Now, the conversion feature sets the value of the `kind` field to `Pipeline` after conversion.
```
Comment (Dhruv-Soni11) [inline modules/op-release-notes-1-10.adoc:210]: This sentence is correct grammatically but I would suggest that you modify this sentence in the "before this update and after this update" template that you have used in the rest of the content to maintain uniformity.

**Comment by abhatt-rh** [comment]
@shipsing confirmed that peer review for this PR is done by @max-cx. Hence adding the `peer-review-done` label

**Pull request merged**